### ISSUE TITLE
restricting to specific auth providers is not working

### ISSUE THREAD

**Issue by johnpapa** [Azure/static-web-apps-cli#114]
I wanted to stop facebook and google as auth providers so I set these in my config file. However, when going to the routes, they still appear. I expected these links not to work, same as in Azure.

```
    {
      "route": "/.auth/login/facebook",
      "statusCode": "404"
    },
    {
      "route": "/.auth/login/google",
      "statusCode": "404"
    },
```

**Comment by manekinekko** [comment]
Oh! That's a regression bug. Thanks for reporting this.

**Comment by manekinekko** [comment]
Good catch @johnpapa. The issue was happening because in the config file, `statusCode` is provided as a String and not Number. I added more checks to prevent that (in #115)

**Comment by manekinekko** [comment]
Closing this issue. Fix landed in a556304
